packages/webawesome/src/components/button/button.css

```
@@ -186,6 +186,9 @@
 .label {
   display: flex;
 }
+:not(.is-icon-button) .label {
+  display: inline-block;
+}
```

```suggestion
.label {
  display: inline-block;
}

.is-icon-button .label {
  display: flex;
}
```

This works perfectly, but mind if we swap the order here? This might better communicate that `inline-block` is the default.